pkgs/development/python-modules/apkid/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, yara-python
+, python3
+}:
+
+buildPythonPackage rec {
+  pname = "apkid";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "rednaga";
+    repo = "APKiD";
+    rev = "v${version}";
+    sha256 = "1p6kdjjw2jhwr875445w43k46n6zwpz0l0phkl8d3y1v4gi5l6dx";
+  };
+
+  propagatedBuildInputs = [
+    yara-python
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  preBuild = ''
+    # Prepare the YARA rules
+    ${python3.interpreter} prep-release.py
```

Comment:
```suggestion
    ${python.interpreter} prep-release.py
```
That will fix it.

Reply:
I don't see the issue with the different interpreter/dependencies after moving to `python3.pkgs.buildPythonApplication`.